instagram-challenge/config/routes.rb

```diff
@@ -0,0 +1,8 @@
+Rails.application.routes.draw do
+  get 'sessions/new' 
+  post  'sessions/new', to: 'sessions#create'
+  get 'users/new'
+  post 'users/new'
+  resources :users
```

Comment:
I think the gets and posts are made redundant by having users as a resources, no?